### DISCUSSION TITLE
How do I negate a statement using MQL with cnquery or cnspec?

### DISCUSSION THREAD

**Discussion by imilchev** [mondoohq/cnquery#153] [Q&A] [answered]
How do I negate statements in MQL?

So far I have this query:
```
k8s.deployment.podSpec["containers"].all(_["image"].contains("tiller"))
```

However, I actually want to check that the `image` does NOT contain `tiller`. Negating with `!` did not work

**Comment by chris-rock** [comment, marked as answer]
You can use `none`, where you say non image in the list should match tiller:

```
k8s.deployment.podSpec["containers"].none(_["image"].contains("tiller"))
```
